docs/site/HasMany-relation.md

```diff
@@ -521,6 +521,8 @@ export class CustomerRepository extends DefaultCrudRepository {
   ];
   ```
 
+  (Notice that the query syntax is a slightly different from LB3)
```

Comment:
Have you considered using an alert -Note or Tip? See https://loopback.io/doc/en/contrib/alerts.html

Personally, I would also include a link to https://github.com/strongloop/loopback-next/issues/3205 to let people know that we are thinking about simplifying the syntax.